mkdocs.yml

```diff
@@ -31,7 +31,9 @@ nav:
     - Usage: test-suite/usage.md
     - Release notes: test-suite/release-notes.md
   - Build-test-deploy bot: bot.md
-  - Pilot repository: pilot.md
+  - Repositories:
+    - EESSI repository: repositories/software_eessi_io.md
```

Comment:
Maybe rename to `repositories/software.eessi.io.md` (so URL will be https://www.eessi.io/docs/repositories/software.eessi.io)